spring-cloud-gateway-webflux/src/test/java/org/springframework/cloud/gateway/webflux/ProductionConfigurationTests.java

```diff
@@ -282,14 +285,13 @@ public void setHome(URI home) {
 			}
 
 			@GetMapping("/proxy/headers")
-			public Mono<ResponseEntity<Map<String, String>>> headers(ProxyExchange<Map<String, String>> proxy) {
+			public Mono<ResponseEntity<Map<String, List<String>>>> headers(ProxyExchange<Map<String, List<String>>> proxy) {
```

Comment:
Man I would have thought this would have just given a comma separate list and not a single value